bin/desi_compute_dark_nonlinear

```diff
@@ -168,7 +168,7 @@ for night in nights:
     nzeros_good = nzeros - args.nskip_zeros
     if nzeros_good < 5:
         log.critical(f'{nzeros} ZEROS on {night} is insufficient when skipping {args.nskip_zeros}')
-        sys.exit(1)
+        continue
```

Comment:
The intention here was that if you don't have enough zeros to make a meaningful nightly bias, it is better to stop (and re-evaluate whether you really need to skip as many zeros at the beginning, etc.) than proceed with an extra noisy nightly bias.  What is the case where you wanted to proceed anyway?

Reply:
discussed with Julien; this is ok